ui/serializers.py

```diff
@@ -149,6 +171,14 @@ def get_is_admin(self, obj):
             return ui_permissions.has_admin_permission(obj, self.context['request'])
```

Comment:
Can you update the docstring for `CollectionSerializer` and `CollectionListSerializer` to say that one is used for the list view and one for the detail view?

Reply:
OK